### PR TITLE
Image alignment depending on ratio

### LINKED ISSUE #1402: Images - ratio-dependent
**Is your feature request related to a problem? Please describe.**
As part of the v1 image work #1319, we need to ensure that images that have portrait or square ratios use the correct grid items. 

Currently our images all use the same grid item. 

**Describe the solution you'd like**
Portrait images should use `layoutGridItemSmall`
Square images should use `layoutGridItemMedium`
Landscape images should use `layoutGridItemLarge`

-`layoutGridItemLarge` This should be used for landscapes.

| Breakpoints | Width |
| ------------ | ------ |
 | <= Group 1 | spans all columns including margin |
 | Group 2 | spans all columns including margin |
 |  Group 3  | spans all columns including margin |
 | Group 4 | spans columns 2-7 out of 8 |
 | >=Group 5  | spans columns 5 - 16 out of 20 |

- `layoutGridItemMedium` This should be used for text. 

| Breakpoints | Width |
| ------------ | ------ |
 | <= Group 1 | spans all columns excluding margins |
 |  Group 2  | spans columns 1-5 out of 6 |
 | Group 3 | spans columns 1-5 out of 6 |
 | Group 4 | spans columns 2-6 out of 8 |
 | >=Group 5  | spans columns  5 - 14 out of 20 |

- `layoutGridItemSmall` This should be used for portrait images. 

| Breakpoints | Width |
| ------------ | ------ |
 | <= Group 1 | spans all columns excluding margins |
 |  Group 2  | spans columns 1-4 out of 6 |
 |  Group 3  | spans columns 1-5 out of 6  |
 | Group 4 | spans columns 2-5 out of 8 |
 | >=Group 5  | spans columns 5 - 12 out of 20 |

**Describe alternatives you've considered**
N/A

**Testing notes**
Confirm that the relevant images (square or portrait) use the correct Grid Layout.

Dev insight: Snapshot jest tests should be added. 
Have a single test article with three images - Landscape, square and portrait. 
Run a Cypress screenshot test at 5 breakpoints that checks for visual regressions in image implementation. 

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by sareh**
Resolves #1402

**Overall change:** Adds different wrappers around the Image depending on the ratio of the image. 

**Code changes:**

-  Add Image Container stories with Ghost Grid
-  Ensure ImageContainer returns correct wrapper depending on image ratio

Screenshots of stories added to storybook - 

Square:
Screen Shot 2019-03-13 at 09 23 13
Portrait:
Screen Shot 2019-03-13 at 09 22 54
Landscape
Screen Shot 2019-03-13 at 09 22 32
Custom ratio:
Screen Shot custom
---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [x] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Review by dr3 — APPROVED** [review]

File: src/app/containers/Image/index.jsx [hunk 1]
```diff
@@ -51,11 +55,20 @@ const ImageContainer = ({ blocks }) => {
   const ratio = (height / width) * 100;
   const rawImageSrc = getRawImageSrc(originCode, locator);
 
+  let Wrapper = GridItemConstrainedLargeNoMargin;
+
+  if (height === width) {
```
Comment (pjlee11) [inline src/app/containers/Image/index.jsx:60]: Double-checking with product whether we want an exact "this is a perfectly square image" (500 x 500) or "this is what the human eye would think is square" (500 x 495) for this line. 

Update: @jimjohnsonrollings confirmed "exactly" square for simplicity of the CMS guidelines

**Comment by pjlee11** [comment]
@sareh Looking good, is there a reason we haven't add any snapshot tests?

**Comment by pjlee11** [comment]
^ Spoke IRL and confirmed that the snapshots would not be useful at this container level

**Review by pjlee11 — APPROVED** [review]

**Comment by ChrisBAshton** [comment]
Tested by changing an image in one of the articles' fixture data to be 1000x1000.

Verified that the correct `layoutGridItemMedium` styling is being applied.

Screen Shot 2019-03-14 at 12 10 37

Test notes in the issue say:

> Dev insight: Snapshot jest tests should be added.

~~The snapshots don't appear to have changed as a result of this. Looks like we've added fixtures but we're only pulling them into Storybook, not our tests 🤔 worth consolidating them and updating our snapshots?~~ EDIT - just saw https://github.com/bbc/simorgh/pull/1409#issuecomment-472812945 and agree we're good to go (updated snapshots would only add snapshots for the container, but not the contents as we inject them in).

NB:

> Have a single test article with three images - Landscape, square and portrait.

That would indeed be nice. @jimjohnsonrollings is it possible to give us a new article asset? Not a blocker for us merging.

> Run a Cypress screenshot test at 5 breakpoints that checks for visual regressions in image implementation.

Blocked on #1249.

**Pull request merged**